Sprint-2/debug/recipe.js

```diff
@@ -12,4 +14,4 @@ const recipe = {
 
 console.log(`${recipe.title} serves ${recipe.serves}
   ingredients:
-${recipe}`);
+${recipe.ingredients}`);
```

Comment:
This is good & works perfectly. Can you please think of accessing each item & format as below?

bruschetta serves 2
ingredients:
olive oil
tomatoes
salt
pepper

Reply:
I've updated that console.log to `console.log(`${recipe.title} serves ${recipe.serves} \ningredients: \n${recipe.ingredients.join('\n')}`);`.
I hope that's what you were looking for.